tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "./tsconfig.base.json",
 	"compilerOptions": {
-		"allowJs": true
+		"allowJs": true,
+		"resolveJsonModule": true
```

Comment:
This is already added in the master branch

Reply:
What? I can't see `resolveJsonModule` on master. Have merged with master only!

Reply:
`tsconfig.base.js`

Reply:
Okay, will remove :)

Reply:
<3